test/evergreen.yml

```diff
@@ -1704,6 +1730,8 @@ buildvariants:
     - name: make-check-test
     - name: unit-test
     - name: fops
+    - name: compile-enable-strict
+    - name: time-shift-sensitivity-test
```

Comment:
The Jenkins job is currently configured to run once per day. If you want to move to a per-commit-merge testing frequency, you may want to trigger a talk with the team.

Reply:
As agreed with the team, the evergreen job will run at per-commit-merge frequency.